### PR TITLE
Standardise Ethereum TOML configuration keys

### PR DESCRIPTION
**Pull request by jplodge-pro**
* Change the `[evm.contracts]` key to instead be `[evm.address]`, as that is the data in the section.
* Updated all usages of said keys in `contracts` and `api`.

### CODE REVIEW

File: packages/contracts/resource/dev.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # blockchain id, e.g. baseSepolia=84532, arbSepolia=421614
 chain_id = 84532
 
-[evm.contracts]
+[evm.address]
 # credbull-devops wallets.  wallet numbers are 1-based (as opposed to 0-based in anvil)
 # devops deployer (wallet 5)
 deployer.private_key = ""
```
Comment (jplodge-pro) [inline packages/contracts/resource/dev.toml:13]: @lucasia Is this key used? Or accidentally left behind?

Comment (lucasia) [reply packages/contracts/resource/dev.toml:13]: it looks safe to be deleted.

i think we can even delete the "dev" environment once we merge in the SDK.  no need to manage so many github secrets anymore.

File: packages/contracts/resource/testnet.toml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 [evm]
 # blockchain id, e.g. baseSepolia=84532, arbSepolia=421614
 chain_id = 84532
 
-[evm.contracts]
+[evm.address]
 # credbull-devops wallets.  wallet numbers are 1-based (as opposed to 0-based in anvil)
 # devops admin/owner (wallet 1) - public address, okay to share
 owner.public_address = "0xD79Be36f61fce3B8EF2FBF22b13B2b9a68eE15A2"
```
Comment (lucasia) [inline packages/contracts/resource/testnet.toml:12]: in addition to the table change, could you also change the address names?  was expecting both:

- table:  [evm.contracts] -> [evm.address]
- addresses, e.g. owner_public_address -> owner

**Review by lucasia — CHANGES_REQUESTED** [review]
in addition to the table change, could you also change the address names? was expecting both:

table: [evm.contracts] -> [evm.address]
addresses, e.g. owner_public_address -> owner

File: packages/contracts/resource/dev.toml (hunk 1 above)
Comment (jplodge-pro) [reply packages/contracts/resource/dev.toml]: Deleted the key.

File: packages/contracts/resource/testnet.toml (hunk 2 above)
Comment (jplodge-pro) [reply packages/contracts/resource/testnet.toml]: Done for `contracts` and `api`.

**Review by lucasia — APPROVED** [review]
looks great, thank you!

**Pull request merged**